tests/exposures/hiv_test.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from conftest import FakeRandom
+
+from titan.exposures import HIV
+from titan.agent import Relationship
+
+
+@pytest.mark.unit
+def test_hiv_expose(make_model, make_agent):
+    model = make_model()
+    model.run_random = FakeRandom(-0.1)  # forces conversion even if prob is 0
```

Comment:
It'd be nice to standardize the comment for `FakeRandom(-0.1)`. Below it's `# always less than param`